app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Admin
+  class OrdersController < ApplicationController
+    def index
+      @orders = {ordered: Order.ordered, paid: Order.paid, completed: Order.completed, cancelled: Order.cancelled}
```

Comment:
Line is too long. [114/80]<br>Space inside { missing.<br>Space inside } missing.